dotcom-rendering/src/web/components/FixedMediumSlowVI.tsx

```diff
@@ -65,11 +59,10 @@ export const FixedMediumSlowVI = ({
 						showDivider={index > 0}
 						containerPalette={containerPalette}
 					>
-						<FrontCard
+						<Card25Media25TallSmallHeadline
```

Comment:
Do you have a screenshot of the 0 sublinks case? This wrapper shows "Trail text when there is no supporting content" according to the description below but I am not sure this is the case. 

Asking because I'm doing something similar for [`FixedLargeSlowXIV`](https://github.com/guardian/dotcom-rendering/pull/6906) and want to check if I can re-use it 😄